src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -107,6 +107,33 @@ std::string MergeInfos(const std::string& first,
                        const std::string& second,
                        const std::string& third);
 
+/**
+ * @brief IsResultCodeUnsupported check if overall result code of provided infos
+ * is UNSUPPORTED_RESOURCE
+ * @param first - contains result_code from HMI response and
+ * interface that returns first response
+ * @param second - contains result_code from HMI response and
+ * interface that returns second response
+ * @return true if overall result code is UNSUPPORTED_RESOURCE otherwise false
+ */
+bool IsResultCodeUnsupported(const ResponseInfo& first,
+                             const ResponseInfo& second);
+
+/**
+ * @brief IsResultCodeUnsupported check if overall result code of provided infos
+ * is UNSUPPORTED_RESOURCE
+ * @param first - contains result_code from HMI response and
+ * interface that returns first response
+ * @param second - contains result_code from HMI response and
+ * interface that returns second response
+ * @param third - contains result_code from HMI response and
+ * interface that returns third response
+ * @return true if overall result code is UNSUPPORTED_RESOURCE otherwise false
+ */
+bool IsResultCodeUnsupported(const ResponseInfo& first,
```

Comment:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/1567/commits/9b902f45d86543319ab7f6aa12375feb5fa18d5f